tutorials/how-to-use-ipfs/index.md

```diff
@@ -0,0 +1,130 @@
+
+```txt
+added Qmbct5cv1SxMpEYCpNEvZnQ5TkN1rk7KyTmV95nbsrbLiR metadata.json
+```
+
+By running `add` you have [pinned the file](https://docs.ipfs.tech/how-to/pin-files/#three-kinds-of-pins) to the node, stopping it from being eventually pruned.
+
+The content identifier (CID) of the file is `Qmbct5cv1SxMpEYCpNEvZnQ5TkN1rk7KyTmV95nbsrbLiR`. This is the CID that you will use to retrieve the file from IPFS.
+
+Verify the content of the file by running:
+
+```sh
+$ ipfs cat Qmbct5cv1SxMpEYCpNEvZnQ5TkN1rk7KyTmV95nbsrbLiR
+```
+
+At this point, you can start your local IPFS node:
+
+```sh
+$ ipfs deamon
+```
+
+The file becomes available on IPFS. You can easily access it via `https://ipfs.io/ipfs/<cid>`, in your case `https://ipfs.io/ipfs/Qmbct5cv1SxMpEYCpNEvZnQ5TkN1rk7KyTmV95nbsrbLiR`.
+
+Note that as soon as you stop the node, it is possible that after a while, the file will not be accessible on the network. This is because only your node has pinned the file, other nodes may delete the file depending of their retention policy.
+
+As you want to store the metadata permanently, without running your IPFS own node (for the scope of this tutorial), you want to pin it to a remote node, using a [pinning service](https://docs.ipfs.tech/how-to/pin-files/#three-kinds-of-pins).
+
+## Pinning services
+
+Pinning services are services that pin files on their nodes, making them available on the network, without having to run your own node.
+
+For this tutorial, you will use [Web3.storage](https://web3.storage/), a pinning service provided by **Protocol Labs**, the main developers of IPFS. Feel free to use any other pinning service you prefer, such as [Pinata](https://pinata.cloud), [Infura](https://infura.io/product/ipfs), etc.
+
+First, create an account on [Web3.storage](https://web3.storage/) and get an [API token](https://web3.storage/tokens/?create=true). With Web3.storage, you need to request access to the pinning API, or you can upload the file directly from the interface.
+
+Next, add `web3.storage` as a remote service:
+
+```sh
+$ ipfs pin remote service add web3.storage https://api.web3.storage/ <YOUR_AUTH_KEY_JWT>
+```
+
+Make sure your node can serve the file when the Web3.storage server looks for it:
+
+```sh
+$ ipfs daemon &
+```
+
+Now you can pin the file to the remote node:
+
+```sh
+$ ipfs pin remote add --service=web3.storage --name="proposal-1-metadata.json" Qmbct5cv1SxMpEYCpNEvZnQ5TkN1rk7KyTmV95nbsrbLiR
+```
+
+Your file is now pinned to the remote node.
+
+## 🎉 Congratulations 🎉
+
+By completing this tutorial, you have learned how to use IPFS for storing files, notably for proposal metadata.
+
+To learn more about IPFS in general, you can check their [documentation](https://docs.ipfs.io/).
+Additionally, you can learn more about how to use the `CID` in the [group](https://tutorials.cosmos.network/tutorials/understanding-group) and the [gov](https://tutorials.cosmos.network/tutorials/understanding-gov) modules tutorials.
```

Comment:
@julienrbrt This understanding-gov is a broken link. Do you know where there is a current version?